server.go

```diff
@@ -1606,6 +1606,40 @@ func (s *Server) applyCreateMeasurementsIfNotExists(m *messaging.Message) error
 	return nil
 }
 
+func (s *Server) DropMeasurement(database, name string) error {
+	c := &dropMeasurementCommand{Database: database, Name: name}
+	_, err := s.broadcast(dropMeasurementMessageType, c)
+	return err
+}
+
+func (s *Server) applyDropMeasurement(m *messaging.Message) error {
+	var c dropMeasurementCommand
+	mustUnmarshalJSON(m.Data, &c)
+
+	database := s.databases[c.Database]
+	if database == nil {
+		return ErrDatabaseNotFound
+	}
+
+	// Remove from metastore.
```

Comment:
This comment implies only the Metastore is affected, but I think there is code in the function below which is also deleting the raw shard data. The comment should make this clear (if I am correct).